src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -576,6 +576,8 @@ void AppCommandlineArgs::_addNewTerminalArgs(AppCommandlineArgs::NewTerminalSubc
         _inheritEnvironment,
         RS_A(L"CmdInheritEnvDesc"));
 
+    subcommand.keepOpenOption = subcommand.subcommand->add_flag("-ko,--keepOpen", _keepOpenOption, RS_A(L"CmdKeepOpenDesc"));

```

Comment:
I would expect that this line also add the new parameter to the help window. But it doesn't do that:
![image](https://github.com/microsoft/terminal/assets/61519853/2925598b-bfc6-4bde-bf68-c2b2b868528e)

Reply:
Ah, yea that's by design. The help text for `wt` is different than the help text for `wt new-tab`. Behind the scenes, we'll secretly treat a `wt` without a subcommand as a `new-tab` subcommand, but the automatically generated help text for CLI11 doesn't know how to handle that 😅